www/chromium/files/patch-components_policy_resources_policy__templates.json

```diff
@@ -1,79 +1,79 @@
---- components/policy/resources/policy_templates.json.orig	2019-09-09 21:55:14 UTC
+--- components/policy/resources/policy_templates.json.orig	2019-10-21 19:06:29 UTC
 +++ components/policy/resources/policy_templates.json
-@@ -1020,7 +1020,7 @@
-       'name': 'HomepageLocation',
+@@ -1038,7 +1038,7 @@
+       'owners': ['file://components/policy/resources/OWNERS'],
        'type': 'string',
        'schema': { 'type': 'string' },
 -      'supported_on': ['chrome.*:8-', 'chrome_os:11-'],
 +      'supported_on': ['chrome.*:8-', 'chrome_os:11-', 'chrome.freebsd:9-'],
```

Comment:
I would guess most people running Chromium on FreeBSD do so on FreeBSD 11+ these days ;)

Reply:
It should be Chrome 9 on FreeBSD, so ancient anyhow. But this just states since when the policy is available.